autotx/tests/agents/token/research/test_research.py

```diff
@@ -1,119 +1,161 @@
-from crewai import Task, Agent
+import os
+from textwrap import dedent
 import pytest
+from autotx.agents import ResearchTokensAgent
 from autotx.agents.ResearchTokensAgent import (
-    ResearchTokensAgent,
     filter_token_list_by_network,
     get_coingecko,
 )
+from autogen import AssistantAgent, UserProxyAgent, GroupChatManager
+import autogen
+
 from autotx.utils.ethereum.networks import ChainId
 
+@pytest.fixture()
+def user_proxy_agent() -> UserProxyAgent:
+    llm_config = { "cache_seed": None, "config_list": [{"model": "gpt-4", "api_key": os.getenv("OPENAI_API_KEY")}]}
+    user_proxy_agent = UserProxyAgent(
+        name="user_proxy_agent",
+        is_termination_msg=lambda x: x.get("content", "") and x.get("content", "").rstrip().endswith("TERMINATE"),
+        human_input_mode="NEVER",
+        max_consecutive_auto_reply=20,
+        system_message=f"You are a user proxy. You will be interacting with the agents to accomplish the tasks.",
+        llm_config=llm_config,
+        code_execution_config=False,
+    )
+    return user_proxy_agent
 
 @pytest.fixture()
-def token_research_agent(auto_tx) -> Agent:
-    return ResearchTokensAgent(auto_tx)
+def manager(auto_tx, user_proxy_agent) -> GroupChatManager:
+    get_llm_config = lambda: { "cache_seed": None, "config_list": [{"model": "gpt-4", "api_key": os.getenv("OPENAI_API_KEY")}]}
```

Comment:
shouldnt we use the env variable `model=os.environ.get("OPENAI_MODEL_NAME", "gpt-4-turbo-preview")`? (like its defined in the `autotx/utils/llm.py` file)

Reply:
Done